### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by jace1084**
updates on Authentication and db insertion. Will be continuing...

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -81,13 +81,14 @@ CREATE TABLE users(
...
 	verified_email BOOLEAN DEFAULT FALSE,
 	-- when inserting into users table the value for email_verification_token should be uuid()
-	email_verification_token VARCHAR(255) NOT NULL,
+	-- Change back to not null when code is further along...
+	email_verification_token VARCHAR(255) DEFAULT NULL,
 	username VARCHAR(30) NOT NULL UNIQUE,
-	first_name VARCHAR(255) NOT NULL,
-	last_name VARCHAR (255) NOT NULL,
-	phone_number VARCHAR(100) NULL,
+	first_name VARCHAR(255) DEFAULT NULL,
```
Comment (simonnguyen3054) [inline db/schema.sql]: @jace1084 Why are these changed to DEFAULT? I think this is going to create a false database if we run the schema.

Comment (jace1084) [reply db/schema.sql]: This is defaulted (for now) due to the dB throwing an error (no default value), therefore I can’t insert other aspects of the signup page unless these values have a default of NULL. This was for testing purposes! I wasn’t planning on merging the the PR yet (that’s why I didn’t assign any reviewers) until I take it out of “testing” mode.

Comment (simonnguyen3054) [reply db/schema.sql]: ok I see. Interesting. Have you re-run your schema again and see if the app still work fine?

File: db/schema.sql [hunk 2]
```diff
@@ -71,7 +71,7 @@ CREATE TABLE userQueries (
 
 CREATE TABLE admin_users (
 	id INT NOT NULL AUTO_INCREMENT,
-	email VARCHAR(255) NOT NULL UNIQUE,
+	email VARCHAR( NOT NULL255) UNIQUE,
```
[the remark this answers is not in the document]
Comment (jace1084) [reply db/schema.sql]: This must be a typo while fixing things with Avanika. I don’t recognize that change and it doesn’t make sense. I’ll change that!
